index.js

```diff
@@ -194,24 +195,24 @@ function purgePlayers(client) {
 }
 
 function convertToByte(num) {
-	num = num & 0xFF;
-	num = num > 127 ? num - 256 : num;
-	return num;
+	num &= 0xFF;
+	new_num = num > 127 ? num - 256 : num;
+	return new_num;
 }
 
 function convertAngle(ang, num) {
-	ang = ang / 256 * Math.PI * 2;
-	ang = (((ang * -1) * RAD2DEG) - (num != undefined ? num : 0)) * 256 / 360;
-	return convertToByte(ang);
+	new_ang = ang / 256 * Math.PI * 2;
+	new_ang = (((ang * -1) * RAD2DEG) - (num != undefined ? num : 0)) * 256 / 360;
+	return convertToByte(new_ang);
```

Comment:
```suggestion
	ang = ang / 256 * Math.PI * 2;
	ang = (((ang * -1) * RAD2DEG) - (num != undefined ? num : 0)) * 256 / 360;
	return convertToByte(ang);
```
reverting unrelated changes not related to the PR v999